modules/apps/frontend-theme-locos/locos-theme/src/css/_banner.scss

```diff
@@ -0,0 +1,93 @@
+%astyle {
+  color: #999990;
+  font-family: 'Poppins', sans-serif;
+}
+%ahover {
+  text-decoration: none;
+  color: #f28959;
+  border-bottom: 3px solid #f28959;
+  -webkit-transition:border-bottom .5s ease;
+  -moz-transition:border-bottom .5s ease;
+  -ms-transition:border-bottom .5s ease;
+  -o-transition:border-bottom .5s ease;
```

Comment:
Do we need all of these vendor prefixes?  If so, is there a bourbon mixin that can take care of all of the vendor prefixes?